dot/parachain/availability-store/availability_store.go

```diff
@@ -456,32 +566,281 @@ func (av *AvailabilityStoreSubsystem) processMessages() {
 				}
 
 			case parachaintypes.ActiveLeavesUpdateSignal:
+				av.currentMessage = msg
 				av.ProcessActiveLeavesUpdateSignal()
 
 			case parachaintypes.BlockFinalizedSignal:
+				av.currentMessage = msg
 				av.ProcessBlockFinalizedSignal()
 
 			default:
-				logger.Error(parachaintypes.ErrUnknownOverseerMessage.Error())
+				// TODO(ed): handle unknown message type (if necessary)
+				if msg != nil {
+					logger.Infof("unknown message type %T", msg)
+					logger.Error(parachaintypes.ErrUnknownOverseerMessage.Error())
+				}
```

Comment:
maybe panic in this case until we can gracefully shut down?

Reply:
Added panic.